account_payment_purchase/readme/USAGE.md

```diff
@@ -0,0 +1,10 @@
+You are able to add a payment mode directly on a partner.
+This payment mode is automatically associated to the purchase order, then on
+related invoice.
+This default value could be change in a draft purchase or draft invoice.
+When you create a payment order, only invoices related to chosen payment mode
+are displayed.
+Invoices without any payment mode are displayed too.
+
+
+This module doesn't add any feature, but it is used by several other modules.
```

Comment:
I still see this text.

Reply:
This is the text in version 16, I have not changed anything:
https://raw.githubusercontent.com/OCA/bank-payment/16.0/account_payment_purchase/readme/USAGE.rst

Reply:
Yes, I know, but now reviewing this migration, I have discovered that it's not correct (both in v16 nor v17), so it's the occasion to fix it, at least in v17.

Reply:
ok, And what would be the correct text?

Reply:
I think the first paragraph is enough. Just remove this one.

Reply:
that's OK?